### ISSUE TITLE
Fail with default heroku-18 stack

### ISSUE THREAD

**Issue by paulcarroty** [jontewks/puppeteer-heroku-buildpack#24]
```
2018-09-15T07:40:54.868060+00:00 app[web.1]: stderr: (node:17) UnhandledPromiseRejectionWarning: Error: Failed to launch chrome!

2018-09-15T07:40:54.868085+00:00 app[web.1]: /app/node_modules/puppeteer/.local-chromium/linux-588429/chrome-linux/chrome: error while loading shared libraries: libpangoft2-1.0.so.0: cannot open shared object file: No such file or directory
```

The libpangoft2 package is needed.

**Comment by nhat-nguyen** [comment]
@paulcarroty I tested [this](https://github.com/nguyenkaos/puppeteer-heroku-buildpack) buildpack from the person who upvoted you and it seemed to work with `heroku-18`. It would be nice if we can upstream the changes here.

**Comment by hilmerx** [comment]
> 2018-09-16T12:46:18.149982+00:00 app[web.1]: /app/node_modules/puppeteer/.local-chromium/linux-579032/chrome-linux/chrome: error while loading shared libraries: libthai.so.0: cannot open shared object file

I  forked the repo and added the "libpangoft2-1.0-0" package, however I get a new dependency error, which is the "libthai.so.0" package which I couldn't find a way to add. And of course, there might be more broken dependencies. I also got this error with both my app and the "https://github.com/timleland/headless-chrome" guide.

Also, the nguyenkaos buildpack referenced above gave me the same dependency error.

**Comment by paulcarroty** [comment]
Thanks, @nguyenkaos, you should make the pull request.
I got it working with latest node, puppeter and `['--no-sandbox', '--disable-setuid-sandbox' ]` flags.

**Comment by hilmerx** [comment]
I tried latest node, puppeteer and the flags. + cleaned cache and still getting the the "libthai.so.0" dependency error. If anyone has any tips how to solve it that'd be very appreciated!

**Comment by paulcarroty** [comment]
The `libthai.so.0` library is a part from [libthai0](https://packages.ubuntu.com/bionic/amd64/libthai0/filelist) package which available in nguyenkaos buildpack.

Connect to your instance via `heroku run bash` and verify this library.

**Comment by hilmerx** [comment]
Thanks a lot @paulcarroty, it's now up and running!

**Comment by jontewks** [comment]
Hey all. Thanks very much for bringing this to my attention and then getting it resolved amongst yourselves!

I did some digging and found that in the heroku-16 stack, those two dependencies listed were available at run time, but on the heroku-18 stack, they moved those two dependencies to build time only. The full list is here: https://devcenter.heroku.com/articles/stack-packages

So I added those in and made a new release for the updates: https://github.com/jontewks/puppeteer-heroku-buildpack/releases/tag/1.1.4

@hilmerx if you would be able to try that release and just confirm its working for you that would be very helpful. I also tested it on heroku-18 just now and all seemed well.

Thanks again for letting me know!
